django/cantusdb_project/main_app/templatetags/helper_tags.py

```diff
@@ -44,3 +44,7 @@ def source_links():
         options += option_str
 
     return mark_safe(options)
+
+@register.filter
+def classname(obj):
```

Comment:
Can you add some comments/doc string to clarify why and where this tag is used? I understand what it does since I'm looking at the changed files side by side, but in the future, this might confuse people. Adding some docs is always helpful.